data/ecosystems/a/ator.toml

```diff
@@ -0,0 +1,57 @@
+# Ecosystem Level Information
+title = "Ator"
+
+sub_ecosystems = []
+
+github_organizations = [
+"https://github.com/iopay/iopay-integration-example"
+]
```

Comment:
This should be a repo and not an organization

Reply:
Sorry, my bad, this was actually a typo, the array should be empty - I fixed it